instrumentation/sofa-rpc/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.alipay.sofa")
+    module.set("sofa-rpc-all")
+    versions.set("[5.0.0,)")
```

Comment:
We use muzzle to check supported range, so the version range should be consistent with that, more information about muzzle, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/muzzle.md